### ISSUE TITLE
a problem when I change the input size and text scale,nvalidArgumentError (see above for traceback): ConcatOp : Dimensions of inputs should match: shape[0] = [14,14,14,2048] vs. shape[1] = [14,13,13,512]

### ISSUE THREAD

**Issue by fujingling** [argman/EAST#213]
Hi,
 I set the input_size=200 ans text_scale=200,cause the size of train image is 200*100,.here is my multigpu_train paremeter

python multigpu_train.py --gpu_list=4 --input_size=200 --batch_size_per_gpu=14 --checkpoint_path=./models/ --text_scale=200 --training_data_path=./datasets/train/ --geometry=RBOX --learning_rate=0.0001 --num_readers=24 

then  I got a error:
InvalidArgumentError (see above for traceback): ConcatOp : Dimensions of inputs should match: shape[0] = [14,14,14,2048] vs. shape[1] = [14,13,13,512]
         [[Node: model_0/feature_fusion/concat = ConcatV2[N=2, T=DT_FLOAT, Tidx=DT_INT32, _device="/job:localhost/replica:0/task:0/gpu:0"](model_0/feature_fusion/ResizeBilinear, model_0/resnet_v1_50/block2/unit_4/bottleneck_v1/Relu, model_0/feature_fusion/concat/axis)]]
         [[Node: model_0/AddN/_1221 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/cpu:0", send_device="/job:localhost/replica:0/task:0/gpu:0", send_device_incarnation=1, tensor_name="edge_28849_model_0/AddN", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/cpu:0"]()]]

**Comment by dsvolkov** [comment]
I suppose that image size should be diveded by 32. So, it can not be 200*100

**Comment by tianmaxingkonggrant** [comment]
@fujingling  did you solved the problem?

**Comment by blaine141** [comment]
I am also having this issue. Making the image size divisible by 32 did not fix it. Even making the image size the exact same as the sample training data did not fix it. @fujingling did you find a solution?
